manifests/cf-manifest/spec/networks_spec.rb

```diff
@@ -1,28 +0,0 @@
-
-RSpec.describe "networks" do
-  CF_NETWORK_NAMES = %w(
-    cf1
-    cf2
-    cf3
-    cell1
-    cell2
-    router1
-    router2
-  )
-
-  let(:networks) { manifest_with_defaults.fetch("networks") }
-
-  CF_NETWORK_NAMES.each do |net_name|
-    describe "#{net_name} network" do
-      let(:network) { networks.find {|n| n["name"] == net_name } }
-
-      it "should have a single subnet" do
-        expect(network["subnets"].length).to eq(1)
-      end
-
-      it "should set the correct subnet ID" do
```

Comment:
No, sorry, it has been moved here: https://github.com/alphagov/paas-cf/blob/cc-move-2/manifests/cf-manifest/spec/cloud-config/networks_spec.rb#L19